frontend/src/components/DrawerCollectionList.tsx

```diff
@@ -45,41 +45,66 @@ const DrawerCollectionListItem: React.FC<DrawerCollectionListItemProps> = observ
 
   return (
     <React.Fragment>
-      <ListItem
-        selected={isSelected(collection.id)}
-        onClick={event => handleListItemClick(event, collection.id)}
-      >
-        <ListItemText primary={collection.name} />
-        {isSelected(collection.id) ? <ExpandLess /> : <ExpandMore />}
-      </ListItem>
+      {(collection.name.length > 32) ? (
+        <Tooltip content={collection.name} direction="right" distance={26}>
+          <ListItem
```

Comment:
This whole ListItem element could be defined once and turned into a function of collection and used here and in L64

Reply:
@pbylicki do you have an idea of how to solve it?
There is a problem with different parameters passed to ListItem and a bit different behavior.
Sorry, I'm not well familiarized with frontend :P

Reply:
@MarcinMaciaszek I would split this new function into two:
`function collectionListItem(item: Collection): JSX.Element`
`function keywordListItem(item: Keyword): JSX.Element`
Otherwise it looks ok. You can also update version number as Maciej asked and it will be ready to merge :)

Reply:
Version bumped, will fix it according to your suggestions with the next ticket.